packages/authenticator/amplify_authenticator/lib/src/state/authenticator_state.dart

```diff
@@ -185,7 +184,8 @@ class AuthenticatorState extends ChangeNotifier {
     notifyListeners();
   }
 
-  Country _country = initialCountryCode;
+  late DialCode _country = defaultDialCode;
```

Comment:
```suggestion
  late DialCode _dialCode = defaultDialCode;
```

You can rename _country as part of these changes so that it is consistently called dial code everywhere.